integration_tests/suite/test_parkings.py

```diff
@@ -0,0 +1,641 @@
+# Copyright 2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+from collections.abc import Generator
+from datetime import datetime, timezone
+from functools import wraps
+from typing import TYPE_CHECKING, Callable, TypedDict, cast
```

Comment:
Not sure which line you are talking about.  Imports should only be in TYPE_CHECKING block to avoid circular imports or if they are not used in runtime and module are not already imported (e.g: annotations only).  Anything from typing should still be imported in the main import statement since the typing module is already imported for the TYPE_CHEKING constant; so in this case even though Callable is only used in annotations, it should still be imported in the main import statement.  

The only that can be moved is Generator, since it is only used in return annotation.  All the others are used by the python interpreter